src/organisms/card-gallery/index.jsx

```diff
@@ -3,16 +3,24 @@ import useStore from "/src/ions/store/index.jsx";
 
 import CardProfile from "../card";
 import CardChip from "../card-chip";
-import GalleryButton from "../../molecules/gallery-button";
-
+import GalleryButton from "/src/molecules/gallery-button";
+import GalleryFilter from "/src/molecules/gallery-filter/index.jsx";
+import { Stack } from "@mui/material";
 const CardGallery = () => {
 	const view = useStore(state => state.view);
+	const cards = useStore(state => state.cards);
+	let filterStatus = useStore(state => state.filterStatus);
+	let filteredCards = useStore(state => state.filteredCards);
```

Comment:
why these are lets? I dont think you can mutate a useStore Hooklike this, you would write a function in your zustand